spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/metadata/JdbcMetadataStore.java

```diff
@@ -245,5 +253,10 @@ public String remove(String key) {
 		}
 		return null;
 	}
-
```

Comment:
Every member in the class must be wrapped with blank lines.
Right now you are sticking the next method to the previous.